src/NexusMods.App.UI/Overlays/MetricsOptIn/MetricsOptInViewModel.cs

```diff
@@ -47,15 +45,15 @@ public MetricsOptInViewModel(ISettingsManager settingsManager, IOverlayControlle
                 HasShownPrompt = true,
             });
 
-            IsActive = false;
+            Close();
         });
     }
 
     public bool MaybeShow()
     {
         if (_settingsManager.Get<TelemetrySettings>().HasShownPrompt) return false;
 
-        _overlayController.SetOverlayContent(new SetOverlayItem(this));
+        _overlayController.Enqueue(this);
         return true;
```

Comment:
Use `OverlayController` from parent class instead of having a private field here.